internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -369,6 +383,7 @@ func (a *ApplicationSnapshotImage) WriteInputFile(ctx context.Context) (string,
 			Ref:        a.reference.String(),
 			Signatures: a.signatures,
 			Config:     a.configJSON,
+			Manifests:  a.manifests,
```

Comment:
I think these manifests should go under something like `olm` (Operator Lifecycle Manager). The term `manifests` on its own is too ambiguous. For example, Red Hat stuffs certain SBOM-like manifests under `/root/buildconfig`.

Reply:
I kinda think we want to be even broader than that, we could go with something like `image.fs`, and keep exposing just the `/manifests/*.(yaml|json)` for now. Once we have the ability to add custom functions to OPA this could be something like `oci.files("registry.io/repository...", ...)`